roles/openshift_cluster_monitoring_operator/files/cluster-monitoring-operator.yaml

```diff
@@ -13,13 +13,17 @@ metadata:
 openshift.io/provider-display-name: Red Hat, Inc.
 parameters:
 - name: OPERATOR_IMAGE
-  value: quay.io/coreos/cluster-monitoring-operator:v0.0.1
+  value: quay.io/coreos/cluster-monitoring-operator:v0.0.6
 - name: PROMETHEUS_OPERATOR_IMAGE
   value: quay.io/coreos/prometheus-operator
 - name: ALERTMANAGER_IMAGE
   value: quay.io/prometheus/alertmanager
+- name: ALERTMANAGER_STORAGE_CAPACITY
+  value: "2Gi"
 - name: PROMETHEUS_IMAGE
   value: quay.io/prometheus/prometheus
+- name: PROMETHEUS_STORAGE_CAPACITY
+  value: "2Gi"
```

Comment:
I think this is too low of a default. We will see lots of people just try the defaults and after a few hours/minutes the disk will be full and we will have tickets opened for it. I’d go with something like 50Gi instead.

Reply:
I thought that too, but then I wondered if the "real" default would be set in openshift-ansible?

https://github.com/openshift/openshift-ansible/pull/8591#issuecomment-393731597

Anyway, after looking at 50G is certainly a reasonable default.

Reply:
Good thoughts- I increased the default to 50Gi and made the fields required in the template.